src/encoded/schemas/quality_metric.json

```diff
@@ -55,7 +56,8 @@
                     "title": "MIME type",
                     "type": "string",
                     "enum": [
-                        "text/plain"
+                        "text/plain",
+                        "image/png"
```

Comment:
Yea I thought it would be more flexible that way.  Limiting it to just text seems equally arbitrary to limiting it to just png.  With both (and the optional size parameters), you get the flexibility of both.

But I may be overlooking other considerations?
Seth

On Feb 2, 2016, at 4:35 PM, Laurence Rowe <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/encoded/schemas/quality_metric.json<https://github.com/ENCODE-DCC/encoded/pull/1136#discussion_r51661735>:

> @@ -55,7 +56,8 @@
>                      "title": "MIME type",
>                      "type": "string",
>                      "enum": [
> -                        "text/plain"
> +                        "text/plain",
> +                        "image/png"


Should every quality_metric allow pngs or should this be decided per type?

—
Reply to this email directly or view it on GitHub<https://github.com/ENCODE-DCC/encoded/pull/1136/files#r51661735>.